### ISSUE TITLE
Is there any offical docker image for sdaps?

### ISSUE THREAD

**Issue by omidraha** [sdaps/docker#1]
Hi,
Is there any offical docker image for sdaps?
Or up to date one  alternative?

**Comment by omidraha** [comment]
May be you can use this one to release an official docker file:
https://github.com/sakalauskas/docker-sdaps

**Comment by benzea** [comment]
No, there is not. See also https://github.com/sdaps/sdaps/issues/112.

We even have the trouble that we would like to get the docker hub account named "sdaps" deleted to make an organisation there … https://hub.docker.com/r/sdaps/
